linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -23,6 +29,9 @@ use tonic_health::pb::{
 };
 use tracing::{info, warn};
 
+/// The endpoint used for all the tests
+const END_TO_END_STORAGE_SERVICE_ENDPOINT: &str = "127.0.0.1:8742";
```

Comment:
If that needs to match the endpoint in the builder, we should use this constant there, too.

Reply:
Yes, that needs to match. And yes it does match. There are indeed two entries `END_TO_END_STORAGE_SERVICE_ENDPOINT` in the file `local_net.rs`.